litefs/proxy.html.md.erb

```diff
@@ -9,10 +9,9 @@ toc: true
 ## Overview
 
 <div class="callout">
-You can only use the built-in HTTP proxy for running LiteFS on Fly.io. Take
-a look at documentation on
-[configuring writes to the primary node](/docs/litefs/getting-started-docker/#configuring-writes-to-primary-node) if you're running LiteFS
-outside of Fly.io!
+You can only use the built-in HTTP proxy described here for running LiteFS on Fly.io. If you're running LiteFS
+outside of Fly.io, you will need to use an alternative method for
+[configuring writes to the primary node](/docs/litefs/getting-started-docker/#configuring-writes-to-primary-node)!
```

Comment:
@andie787 I think this place is the only one where you actually should check. The other changes are literally just your suggestion. 😁

Reply:
I'm sorry I didn't see this sooner! This is an example of a beautiful inline link. :)